frontend/apps/pablo/components/Organisms/XPablosBox.tsx

```diff
@@ -48,52 +53,72 @@ export const XPablosBox: React.FC<XPablosBoxProps> = ({
   const expired = (expiry: number) => expiry < new Date().getTime();
 
   return (
-    <BoxWrapper
-      title={title || "Your xPBLO"}
-      {...boxProps}
-    >
+    <BoxWrapper title={title || "Your xPBLO"} {...boxProps}>
       <TableContainer>
         <Table>
           <TableHead>
             <TableRow>
-              {
-                (header || tableHeaders).map((th) => (
-                  <TableCell key={th.header} align="left">{th.header}</TableCell>
-                ))
-              }
+              {(header || tableHeaders).map((th) => (
+                <TableCell key={th.header} align="left">
+                  {th.header}
+                </TableCell>
+              ))}
             </TableRow>
           </TableHead>
           <TableBody>
-            {xPablos.map(({tokenId, locked, expiry, multiplier, amount}: XPablo) => (
-              <TableRow key={tokenId}>
-                <TableCell align="left">
-                  <BaseAsset
-                    icon={getToken(tokenId).icon}
-                    label={getToken(tokenId).symbol}
-                  />
-                </TableCell>
-                <TableCell align="left">
-                  <Typography variant="body1">{locked.toFormat(2)}</Typography>
-                </TableCell>
-                <TableCell align="left">
-                  <Typography
-                    variant="body1"
-                    color={expired(expiry) ? "error" : undefined}
-                  >
-                    {expired(expiry)
-                      ? "Expired"
-                      : moment(expiry).utc().format("DD MMM YYYY")
-                    }
-                  </Typography>
-                </TableCell>
-                <TableCell align="left">
-                  <Typography variant="body1">{multiplier.toFixed(2)}</Typography>
-                </TableCell>
-                <TableCell align="left">
-                  <Typography variant="body1">{amount.toFormat(2)}</Typography>
-                </TableCell>
-              </TableRow>
-            ))}
+            {xPablos.map(
+              (
+                { tokenId, locked, expiry, multiplier, amount }: XPablo,
+                index: number
+              ) => (
+                <TableRow key={tokenId + index}>
```

Comment:
NIT
This table seems like it might be difficult to maintain, would it be possible to add comments above each cell expressing what the content is?

Reply:
Yup, but since this was not a major refactor I haven't changed it. it is nice to come back to this.